tests/SampleProjects/SampleConsoleApp_2.19.x/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright 2021-present MongoDB Inc.
```

Comment:
nit: I didn't look at other files, but in the driver we use a bit different formatting: `/**/`